### PR TITLE
Feat / Documentation sprint 3

### PR DESCRIPTION
**Pull request by langemike**
Improved docs.

1. Cherry picked [6af187a](https://github.com/Videodock/ott-web-app/pull/56/commits/6af187af80e9ce199e9a5ad61cdfea78ba0d559c) from PR #56 
2. Feedback processed from the original PR (mostly suggestions from Roy)
3. Added inline comments for `.env` variables (I tried to keep it simple)
4. Added documentation for `yarn generate-pwa-assets`

Ticket: https://videodock.atlassian.net/browse/OTT-891

### CODE REVIEW

File: platforms/web/.env
Comment (langemike) [inline platforms/web/.env]: Should we remove this line / explanation? I think it's incorrect information.

**Review by VincentCouzij — APPROVED** [review]

**Review by royschut — APPROVED** [review]

**Comment by langemike** [comment]
@ChristiaanScheermeijer I'm awaiting your feedback or approval before I merge :D

**Comment by ChristiaanScheermeijer** [comment]
Nice @langemike! Let's PR this separately from the sprint-3 branch to the open-source repo. That will make reviewing and applying changes isolated from all the features and fixes easier.

**Pull request closed without merging**
